app/controllers/goodsRecord/CyaUpdateRecordController.scala

```diff
@@ -259,19 +250,7 @@ class CyaUpdateRecordController @Inject() (
     putRecordRequest: PutRecordRequest
   )(implicit hc: HeaderCarrier): Future[Done] =
     if (newValue != oldValue) {
-
-      // TODO: remove this flag when EIS has implemented the PATCH method - TGP-2417 and keep the call to putGoodsRecord as default
-      if (config.useEisPatchMethod) {
-        goodsRecordConnector.putGoodsRecord(
-          putRecordRequest,
-          newUpdateGoodsRecord.recordId
-        )
-      } else {
-        goodsRecordConnector.updateGoodsRecord(
-          newUpdateGoodsRecord
-        )
-      }
-
+      goodsRecordConnector.putGoodsRecord(putRecordRequest, newUpdateGoodsRecord.recordId)
```

Comment:
i feel like maybe we could combine these two funcs for readability since they are virtually the same - just an idea